pom.xml

```diff
@@ -45,7 +45,7 @@
 		<version.javadoc.plugin>3.1.0</version.javadoc.plugin>
 		<version.jackson>2.8.0</version.jackson>
 		<version.httpcore>4.4.6</version.httpcore>
```

Comment:
I see that fluent-hc 4.5.13 depends on httpcore 4.4.13, do you think it's reasonable to update it, too?

Reply:
Fixed in a907e7d.